content/api/hub/sample_jinja2.md

```diff
@@ -2,7 +2,7 @@ title = "Jinja2 Template Engine and Spin"
 template = "render_hub_content_body"
 date = "2023-12-08T00:00:00Z"
 content-type = "text/html"
-tags = ["python", "templating"]
+tags = ["python"]
```

Comment:
`templating` seems reasonable to me for an item described as "template engine".  Are you concerned it's too close to the "template" category?

Reply:
Reverted